devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -319,7 +369,23 @@ public void ClearDirectAligned(int x, int y, int width, int height)
         /// <param name="inverted">Indicates if color to be used turn the pixel on, or leave off.</param>
         public void DrawPixel(int x, int y, bool inverted = true)
         {
-            if ((x >= Width) || (y >= Height))
+            switch (Rotation)
+            {
+                case 1:
```

Comment:
And here, use the enum rather than the numbers.

Reply:
Done